apps/web/netlify.toml

```diff
@@ -20,6 +20,30 @@
   status = 301
   force = true
 
+[[redirects]]
+  from = "https://web.novu.co/*"
+  to = "https://dashboard.novu.co/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "http://web.novu.co/*"
```

Comment:
It's interesting/surprising that we need an `http` redirect as well and they don't always redirect to HTTPS by default or using HSTS

Reply:
this was mentioned in the netlify docs that http and https need separate rules. I've just followed the instructions without too much thought